addons/survey/static/src/xml/survey_choice_image_zoomer_templates.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="survey.survey_choice_image_zoomer">
+        <div role="dialog" class="o_survey_img_zoomer_modal modal o_technical_modal fade d-flex align-items-center p-0"
+            aria-labbelledby="Image Zoom Dialog" tabindex="-1">
```

Comment:
To double-check if `o_technical_modal` and `tabindex="-1"` are necessary.

Reply:
tabindex is needed to use the esc key to close the modal :)